addons/event_sale/models/event.py

```diff
@@ -41,6 +43,17 @@ class Event(models.Model):
         'event.event.ticket', 'event_id', string='Event Ticket',
         copy=True)
 
+    sale_order_lines_ids = fields.One2many(
+        'sale.order.line', 'event_id',
+        string='All sale order lines pointing to this event')
+
+    sales_total_price = fields.Monetary(compute='_compute_sales_total_price')
+    currency_id = fields.Many2one(
+        'res.currency', string='Currency',
+        default=lambda self: self.env.company.currency_id.id, readonly=True)
+
+    start_sale_date = fields.Date('Start sale date', compute='_compute_start_sale_date')
```

Comment:
Don't you need the start_date_end one to display somewhere ? Just asking.

Reply:
@tde-banana-odoo no :) we need this date to show in when we cannot yet buy tickets on website. I need the `deadline` of the `event.ticket`, but not the "start_date_end" of the event itself :)